apps/maestro/src/server/routers/interchainToken/recordInterchainTokenDeployment.ts

```diff
@@ -97,7 +102,10 @@ export const recordInterchainTokenDeployment = protectedProcedure
         let tokenAddress;
         let tokenManagerAddress;
 
-        if (!axelarChainId.includes("sui")) {
+        if (
+          !axelarChainId.includes("sui") &&
+          !axelarChainId.includes("stellar")
+        ) {
```

Comment:
Could you please refactor this by using the `supportWagmi` boolean instead? (and below as well)